editor/src/messages/portfolio/document/node_graph/node_graph_message_handler/node_properties.rs

```diff
@@ -512,6 +512,33 @@ pub fn blend_properties(document_node: &DocumentNode, node_id: NodeId, _context:
 	vec![backdrop, blend_mode, LayoutGroup::Row { widgets: opacity }]
 }
 
+pub fn output_properties(_document_node: &DocumentNode, _node_id: NodeId, context: &mut NodePropertiesContext) -> Vec<LayoutGroup> {
+	// TODO: Disable the download/copy buttons if the layer's graph does not output an ImageFrame
```

Comment:
@TrueDoctor I'm not sure how to disable this only when a layer is outputting something other than an `ImageFrame` but it's relatively important to make that happen. Help would be appreciated! Also @0HyperCube if you happen to be able to help out before True Doctor, thanks 😄